frontend/components/TableContainer/DataTable/SoftwareNameCell/SoftwareNameCell.tsx

```diff
@@ -12,6 +12,18 @@ import TooltipTruncatedTextCell from "../TooltipTruncatedTextCell";
 
 const baseClass = "software-name-cell";
 
+/**
```

Comment:
Let's move this into `frontend/pages/SoftwarePage/helpers.tsx` and export it.

Then we can import it into each of the use cases. <3